### PR TITLE
Add compromised/breached collection.

### PR DESCRIPTION
**Pull request by EvgeniyMeteliza**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Add compromised/breached to the list of available commands and to the fetch incidents command and all stuff tied to the new incident type

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by marcellarichmond** [comment]
@mchasepan

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py [hunk 1]
```diff
@@ -886,7 +927,14 @@ def get_info_by_id_for_collection(client: Client, args: Dict) -> List[CommandRes
                 result["indicatorToolRelationships"], result["indicatorsIds"], \
                 result["indicators"]
 
-        main_table_data, additional_tables = transform_function(result)
+        if coll_name == "compromised/breached":
+            if "updateTime" in result:
+                del result["updateTime"]
+
+        if coll_name == "compromised/breached":
+            main_table_data, additional_tables = result, []
```
Comment (Bargenish) [inline Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py:935]:
```suggestion
                del result["updateTime"]
            main_table_data, additional_tables = result, []
```

**Review by Bargenish — CHANGES_REQUESTED** [review]
@EvgeniyMeteliza Thank you for your contribution.

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+#### Classifiers
+##### Group-IB Threat Intelligence & Attribution (classifier)
+- Added **GIB Data Breach** to classifier.
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md]: Done

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- **GIB Data Breach**
+
+#### Integrations
+##### Group-IB Threat Intelligence & Attribution
+- Added **GIB Data Breach** processing to integration.
+- Added new search by id command for compromised/breached collection.
+
+#### Layouts
+##### New: GIB Data Breach Layout
+- Added layout for **GIB Data Breach**.
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md]: Done

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+#### Mappers
+##### Group-IB Threat Intelligence & Attribution (mapper)
+- Added **GIB Data Breach** to mapper.
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md]: Done

File: Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- **GIB ID**
+- **GIB Password**
+
+#### Incident Types
+- **GIB Data Breach**
+
+#### Integrations
+##### Group-IB Threat Intelligence & Attribution
+- Added **GIB Data Breach** processing to integration.
+- Added new search by id command for compromised/breached collection.
```
[the remark this answers is not in the document]
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_1_0.md]: Done

File: Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py (hunk 1 above)
Comment (EvgeniyMeteliza) [reply Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py]: Done

**Review by Bargenish — APPROVED** [review]

**Comment by Bargenish** [comment]
There's an issue with the API.
Will merge the PR once I'll get an update from @EvgeniyMeteliza

**Pull request merged**